<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolClass</class>
 <widget class="QMainWindow" name="ToolClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Tool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>130</y>
      <width>419</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>法線マップ計算色</string>
    </property>
    <widget class="QRadioButton" name="alpha">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>86</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>アルファ</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="red">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>86</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>赤</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="luminance">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>輝度(R、G、Bから計算された単色値)</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="blue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>86</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>青</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="green">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>86</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>緑</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="default_radio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>86</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>デフォルト</string>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>1261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>419</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>テクスチャ座標関連(不明)</string>
    </property>
    <widget class="QCheckBox" name="mirror_u">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>テクスチャのU座標反転</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="mirror_v">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>テクスチャのV座標反転</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>419</width>
      <height>115</height>
     </rect>
    </property>
    <property name="title">
     <string>振幅(法線のスケーリング係数)</string>
    </property>
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-999.990000000000009</double>
     </property>
     <property name="maximum">
      <double>999.990000000000009</double>
     </property>
     <property name="value">
      <double>-15.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>符号 
   + OpenGL用
   - DirectX用</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>419</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>その他処理</string>
    </property>
    <widget class="QCheckBox" name="invert_vector">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>計算された法線ベクトルの符号を反転</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="compute_occlusion">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>オクルージョン計算
(未処理のオクルージョンを計算し、結果をアルファチャネルに格納します。)</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>581</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>compute (ctrl+b)</string>
    </property>
   </widget>
   <widget class="QOpenGLWidget" name="openGLWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>9</y>
      <width>837</width>
      <height>621</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>プレビュー画像</string>
    </property>
    <widget class="QRadioButton" name="diffuse_change">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ディフューズマップ</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="diffuse_change_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ノーマルマップ</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>初期設定に戻す (ctrl+r)</string>
    </property>
   </widget>
   <zorder>groupBox_5</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>pushButton</zorder>
   <zorder>progressBar</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>openGLWidget</zorder>
   <zorder>pushButton_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Tool.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
